tartufo/scanner.py

```diff
@@ -499,7 +492,44 @@ def calculate_entropy(self, data: str) -> float:
             entropy += -probability * math.log2(probability)
         return entropy
 
-    def scan(self) -> Generator[Issue, None, None]:
+    @property
+    def issue_file(self) -> IO:
+        if not self._issue_file:
+            self._issue_file = (
+                tempfile.NamedTemporaryFile(  # pylint: disable=consider-using-with
+                    dir=self.global_options.temp_dir
+                )
+            )
+        return self._issue_file
+
+    def load_issues(self) -> Generator[Issue, None, None]:
+        # Rewind the issue_file
+        self.logger.debug("Rewinding pickle file")
+        self.issue_file.seek(0)
+        while True:
+            try:
+                length = int.from_bytes(self.issue_file.read(4), "little")
+                buf = self.issue_file.read(length)
+                issue = pickle.loads(gzip.decompress(buf))
+                yield from issue
```

Comment:
Can we rename this to `issues` or `issue_list`? Up until I read how `store_issue` is working, I was assuming this was an individual issue, and wondering why in the world we would `yield from` a single item instead of just `yield`ing it.